src/bluesky/protocols.py

```diff
@@ -188,9 +188,21 @@ def get_index(self) -> SyncOrAsync[int]:
 
 
 @runtime_checkable
-class Configurable(Protocol[T]):
+class Configurable(Protocol):
```

Comment:
this drops the generics which I suspect causes a run-time typecheck to fail and not call the right methods.